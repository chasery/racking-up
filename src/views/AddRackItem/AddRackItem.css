.AddRackItem {
  padding-top: 4.5rem;
}

.AddRackItem__wrapper {
  display: flex;
  justify-content: center;
  padding: 1rem 1.5rem;
}

@media all and (min-width: 50rem) {
  .AddRackItem__wrapper {
    max-width: 64rem;
    align-self: center;
    margin: 0 auto;
  }
}
