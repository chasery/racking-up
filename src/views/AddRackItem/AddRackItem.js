import React from "react";
import Form from "../../components/Form/Form";
import FormField from "../../components/FormField/FormField";
import "./AddRackItem.css";

function AddRackItem(props) {
  return (
    <main role="main">
      <section className="AddRackItem">
        <div className="AddRackItem__wrapper">
          <Form id="AddRackItem">
            <h2>Add Rack Item</h2>
            <p>
              Enter the information for the clothing item you would like to add.
            </p>
            <FormField id="name" label="Name" type="text" isRequired={true} />
            <FormField id="url" label="URL" type="text" isRequired={true} />
            <FormField id="price" label="Price" type="text" isRequired={true} />
            <button>Cancel</button>
            <button>Add Rack Item</button>
          </Form>
        </div>
      </section>
    </main>
  );
}

export default AddRackItem;
