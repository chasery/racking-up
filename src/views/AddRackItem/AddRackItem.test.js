import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import AddRackItem from "./AddRackItem";

describe("AddRackItem component", () => {
  it("renders without error", () => {
    const wrapper = shallow(<AddRackItem />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
