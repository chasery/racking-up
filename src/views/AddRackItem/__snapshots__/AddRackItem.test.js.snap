// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddRackItem component renders without error 1`] = `
<main
  role="main"
>
  <section
    className="AddRackItem"
  >
    <div
      className="AddRackItem__wrapper"
    >
      <Form
        id="AddRackItem"
      >
        <h2>
          Add Rack Item
        </h2>
        <p>
          Enter the information for the clothing item you would like to add.
        </p>
        <FormField
          id="name"
          isRequired={true}
          label="Name"
          type="text"
        />
        <FormField
          id="url"
          isRequired={true}
          label="URL"
          type="text"
        />
        <FormField
          id="price"
          isRequired={true}
          label="Price"
          type="text"
        />
        <button>
          Cancel
        </button>
        <button>
          Add Rack Item
        </button>
      </Form>
    </div>
  </section>
</main>
`;
